clmm/models/__init__.py

```diff
@@ -1,2 +1,3 @@
+"""Modules that build off of the Model superclass"""
 from .model import *
```

Comment:
Let's call this a temporary fix.  But, we will want to avoid this in the future to prevent potential namespace conflicts and also to make it easier to read through the code and trace through.